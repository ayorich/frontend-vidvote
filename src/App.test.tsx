import React from 'react';
import { render } from '@testing-library/react';
// import App from './App';

test('renders app component', () => {
    const { asFragment } = render(<div>Hello</div>);
    expect(asFragment()).toMatchSnapshot();
});
