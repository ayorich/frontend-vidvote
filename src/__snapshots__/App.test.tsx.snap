// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders app component 1`] = `
<DocumentFragment>
  <div>
    Hello
  </div>
</DocumentFragment>
`;
